src/main/java/org/fogbowcloud/saps/engine/core/scheduler/arrebol/Arrebol.java

```diff
@@ -20,11 +21,13 @@
 
 	public List<JobSubmitted> returnAllJobsSubmitted();
 
-	public JobResponseDTO checkStatusJobById(String jobId) throws GetJobException;
+	public JobResponseDTO checkStatusJobById(String jobId)
+        throws GetJobException, ArrebolConnectException;
 	
 	public List<JobResponseDTO> checkStatusJobByName(String JobName) throws GetJobException;
 
-	public String checkStatusJobString(String jobId) throws GetJobException;
+	public String checkStatusJobString(String jobId) throws GetJobException, ArrebolConnectException;
```

Comment:
Apparently this method is dead, check if it is still used in any case, if you do not remove it